packages/app/components/token-picker/TokenPickerListRow.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import { BodyText } from "@/ui";
+import { TokenIcon } from "../TokenIcon";
+import { Token } from "@/models/token/types";
+
+interface TokenPickerListRowProps {
+  token: Token;
+  closeAction: () => void;
+}
+
+export const TokenPickerListRow = ({
+  closeAction,
+  token
+}: TokenPickerListRowProps) => (
+  <div
+    className="flex justify-between w-full px-6 py-2 cursor-pointer hover:bg-surface-50"
+    key={`${token.address}${token.symbol}`}
+    onClick={closeAction}
+  >
+    <div className="flex items-center justify-between w-full">
+      <div className="flex items-center space-x-3">
+        <TokenIcon token={token} className="w-8 h-8" />
```

Comment:
I think we can use  token icon prop` size="md"` here.

```suggestion
        <TokenIcon token={token} size="md" />
```

Reply:
You're right! Good catch c: